### ISSUE TITLE
Delete `gh-readonly-queue` branches

### ISSUE THREAD

**Issue by BD103** [bevyengine/bevy#11655]
There are a slew of branches on the main repository prefixed with `gh-readonly-queue`. These branches appear to be left over from Github's merge queue bot. The newest of these branches is 8 months old, so I believe the bot is no longer creating these spare branches (or now deletes them afterward).

If these branches are no longer necessary, can they be deleted?

<img width="1552" alt="image" src="https://github.com/bevyengine/bevy/assets/59022059/ef87e4cc-f8bc-485c-aa3b-f150bdfcc6c6">

**Comment by BD103** [comment]
There are also many other stale branches that could also be deleted, but those are technically out of scope of this issue.

- `apply-deferred`
- `stray-cargo`
- `create-pull-request/patch`
- `gh-pages` (unused after #10892)
- `debug-framecount`

**Comment by alice-i-cecile** [comment]
I've cleaned up the incidental "I made a branch on the main repo" branches now, but most of the `gh-read-only-queue` branches could not be deleted. I'm not sure why; I'll file a ticket with customer support.

**Comment by BD103** [comment]
Awesome! It's weird that you could not delete them. Did you try using Github on the web or through `git`?

Additionally, did `gh-pages` not let you delete it, or are you keeping it for another reason?

**Comment by alice-i-cecile** [comment]
On the web. I wasn't confident about gh-pages so I left it for now.

**Comment by alice-i-cecile** [comment]
Support update:

> Thank you for bringing this issue to our attention. I have escalated this to engineering to remove the lock from this branch and will let you know as soon as I have an update from them.

**Comment by alice-i-cecile** [comment]
![image](https://github.com/bevyengine/bevy/assets/3579909/8343081f-edd6-410b-a2c7-650fbee7e3cb)

Customer support has come through for me: these are now deleted. Thanks!
